server/src/test/java/org/elasticsearch/cli/MultiCommandTests.java

```diff
@@ -102,4 +126,27 @@ public void testSubcommandArguments() throws Exception {
         assertFalse(output, output.contains("command1"));
         assertTrue(output, output.contains("Arguments: [foo, bar]"));
     }
+
+    public void testClose() throws Exception {
+        DummySubCommand subCommand1 = new DummySubCommand();
+        DummySubCommand subCommand2 = new DummySubCommand();
+        multiCommand.subcommands.put("command1", subCommand1);
+        multiCommand.subcommands.put("command2", subCommand2);
+        multiCommand.close();
+        assertTrue("MultiCommand must have been closed when close method is invoked", multiCommand.closed.get());
```

Comment:
for me `must have been closed` points me in the wrong direction; I'd use `should have been` or `was not closed`